src/main/resources/labels/labels.json

```diff
@@ -5671,6 +5671,96 @@
         "multilangLabel": { }
     },
     {
+        "uri": "http://purl.org/lobid/lv#contributorOrder",
+        "label": "contributorOrder",
```

Comment:
Let's have a look at the original branch `72-MARCRelatorCodesInMABRDA` and fix that one.